airflow/providers/openlineage/plugins/adapter.py

```diff
@@ -117,8 +118,10 @@ def emit(self, event: RunEvent):
             self._client = self.get_or_create_openlineage_client()
         event = self._redacter.redact(event, max_depth=20)
         try:
-            return self._client.emit(event)
+            with Stats.timer("ol.emit.attempts"):
```

Comment:
@harels, this is great. It will help with the telemetry open lineage events.
Would it make sense for us to add or modify an existing test to assert we're sending these to Statsd?

Reply:
+1, yes a test would help here.

Reply:
+1. Test should also include scenario for exception